### PR TITLE
fixed some more documentation warnings

### PR DESCRIPTION
**Pull request by jay1723**
**What kind of change does this PR introduce?**
  - [ ] bug fix
  - [ ] feature
  - [x] docs update
  - [ ] tests/coverage improvement
  - [ ] refactoring
  - [ ] other

**What is the related issue number (starting with `#`)**

#1797

**What is the current behavior?** (You can also link to an open issue here)

cherrypy/lib/cptools.py: Warning on block indents and unexpected unindents.
docs/advanced.rst: Warning on newline in comment
docs/basic.rst: Warning for 'memecached' duplicated 

**What is the new behavior (if this is a feature change)?**

Fixed warnings

**Other information**:

N/A

**Checklist**:

  - [ ] I think the code is well written
  - [ ] I wrote [good commit messages][1]
  - [ ] I have [squashed related commits together][2] after the changes have been approved
  - [ ] Unit tests for the changes exist
  - [ ] Integration tests for the changes exist (if applicable)
  - [ ] I used the same coding conventions as the rest of the project
  - [ ] The new code doesn't generate linter offenses
  - [ ] Documentation reflects the changes
  - [ ] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences

[1]: http://chris.beams.io/posts/git-commit/
[2]: https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

### CODE REVIEW

**Comment by webknjaz** [comment]
@jay1723 try doing the following:
```console
git config --global user.name '<YOUR FULL NAME HERE>'
git config --global user.email '<YOUR PROPER EMAIL HERE>'
git commit --amend --reset-author
git push --force-with-lease your_feature_branch:your_feature_branch
```

Oh, and also using the master branch in your fork is highly discouraged because you can create a lot of mess when making more than one PR at a time.

File: docs/basics.rst [hunk 1]
```diff
@@ -579,7 +579,7 @@ it is distributed and a good choice if you want to
 share sessions outside of the process running CherryPy.
 
 Requires that the Python
-`memcached <https://pypi.org/project/memcached>`_
+`python memcached <https://pypi.org/project/memcached>`_
```
Comment (webknjaz) [inline docs/basics.rst]: Why did you do this? Now you have two `Python python` in a row.

File: cherrypy/lib/cptools.py [hunk 2]
```diff
@@ -411,13 +411,8 @@ def session_auth(**kwargs):
...
 
 session_auth.__doc__ = (
-    """Session authentication hook.
-
-    Any attribute of the SessionAuth class may be overridden via a keyword arg
-    to this function:
-
-    """ + '\n'.join(['%s: %s' % (k, type(getattr(SessionAuth, k)).__name__)
-                     for k in dir(SessionAuth) if not k.startswith('__')])
+    "Session authentication hook.\n" +  "Any attribute of the SessionAuth class may be overridden via a keyword arg to this function:"
```
Comment (webknjaz) [inline cherrypy/lib/cptools.py]: This change is superseded by #1812, we don't need it in this PR. Also, you don't need an explicit string concatenation in between brackets + there's two many spaces after the operator.

File: docs/advanced.rst [hunk 3]
```diff
@@ -568,8 +568,7 @@ to integrate common database interfaces such as the DB-API specified in
 <http://sqlalchemy.readthedocs.org>`_ or `SQLObject
 <https://pypi.python.org/pypi/SQLObject/>`_.
 
-You will find a recipe at `cherrypy-recipes
- <https://bitbucket.org/Lawouach/cherrypy-recipes/src/tip/web/database/sql_alchemy/>`_
+You will find a recipe at `cherrypy-recipes <https://bitbucket.org/Lawouach/cherrypy-recipes/src/tip/web/database/sql_alchemy/>`_
```
Comment (webknjaz) [inline docs/advanced.rst]: Long lines are harmful. We don't need this change.

**Comment by webknjaz** [comment]
Please post the log excerpt with the warning that you're trying to address.

**Comment by flamableconcrete** [comment]
I believe this PR can be closed in lieu of #1823

**Comment by webknjaz** [comment]
@jay1723
Thanks, Jay!

**Pull request merged**